OnlineStore/store/pom.xml

```diff
@@ -33,6 +33,23 @@
             <artifactId>javafaker</artifactId>
             <version>1.0.2</version>
         </dependency>
+
+        <dependency>
```

Comment:
please validare groupId and artifactId of your project, they should follow maven naming convention